.github/workflows/test.yaml

```
@@ -365,4 +365,4 @@ jobs:
         if: failure()
         with:
           name: kwok-logs-${{ github.run_id }}-${{ matrix.os }}-${{ matrix.kwokctl-runtime }}${{ matrix.nerdctl-version && '-nerdctl' || '' }}${{ matrix.nerdctl-version }}
```

```suggestion
          name: kwok-logs-${{ github.run_id }}-${{ matrix.os }}-${{ matrix.kwokctl-runtime }}${{ matrix.nerdctl-version && format('-nerdctl-{0}', matrix.nerdctl-version) || '' }}
```

https://docs.github.com/en/actions/learn-github-actions/expressions#format